core/lib/zksync_core/Cargo.toml

```diff
@@ -39,6 +39,12 @@ zksync_health_check = { path = "../health_check" }
 vlog = { path = "../vlog" }
 
 multivm = { path = "../multivm" }
+# Consensus dependenices
+zksync_concurrency = { package = "concurrency", version = "0.1.0", git = "https://github.com/matter-labs/era-consensus.git", rev = "a0905d4896a12d60296e24130dee6a0c0b730c32" }
+zksync_consensus_schema = { package = "schema", version = "0.1.0", git = "https://github.com/matter-labs/era-consensus.git", rev = "a0905d4896a12d60296e24130dee6a0c0b730c32" }
+zksync_consensus_roles = { package = "roles", version = "0.1.0", git = "https://github.com/matter-labs/era-consensus.git", rev = "a0905d4896a12d60296e24130dee6a0c0b730c32" }
+zksync_consensus_storage = { package = "storage", version = "0.1.0", git = "https://github.com/matter-labs/era-consensus.git", rev = "a0905d4896a12d60296e24130dee6a0c0b730c32" }
+zksync_consensus_executor = { package = "executor", version = "0.1.0", git = "https://github.com/matter-labs/era-consensus.git", rev = "a0905d4896a12d60296e24130dee6a0c0b730c32" }
```

Comment:
When are we changing package names in the consensus repo, btw?

Reply:
https://github.com/matter-labs/era-consensus/pull/25